### ISSUE TITLE
Can I use it in jenkins pipeline？

### ISSUE THREAD

**Issue by JermaHuang** [gesellix/docker-client#40]
I want to make a pipeline job to pull  images and create  containers on remote docker。jenkins‘s pipeline is written in groovy，so can i import the api into a jenkins pipeline job

**Comment by gesellix** [comment]
Yes, you can, the client isn't tied to any specific platform or environment. If you struggle to make it work, just check back here with more details!

**Comment by JermaHuang** [comment]
Thanks very much，it's appreciate for your comfirm，i'll try it with the docs
